src/routes/index.js

```diff
@@ -38,11 +38,13 @@ app.get('/', (request, response) => {
 
 app.get('/complete', (request, response) => {
   getCompletedTodos()
-  .then((task) => {
-    response.render('completed', { task })
+  .then((tasks) => {
+    // console.log('BOB :: =>', tasks)
+    response.render('completed', { tasks })
```

Comment:
above you named the array of rows returned `incompletes` so I would name this one `completes` or better yet, name the one above `uncompletedTasks` and this variable `completedTasks`